lib/jsdom/living/xhr/XMLHttpRequest-impl.js

```diff
@@ -83,945 +80,883 @@ const corsSafeResponseHeaders = new Set([
 
-      value = normalizeHeaderValue(value);
+    if (!tokenRegexp.test(header) || !fieldValueRegexp.test(value)) {
+      throw DOMException.create(this._globalObject, [
+        "The string did not match the expected pattern.",
+        "SyntaxError"
+      ]);
+    }
 
-      if (!tokenRegexp.test(header) || !fieldValueRegexp.test(value)) {
-        throw DOMException.create(this._globalObject, [
-          "The string did not match the expected pattern.",
-          "SyntaxError"
-        ]);
-      }
+    const lcHeader = header.toLowerCase();
 
-      const lcHeader = header.toLowerCase();
+    if (forbiddenRequestHeaders.has(lcHeader) || lcHeader.startsWith("sec-") || lcHeader.startsWith("proxy-")) {
+      return;
+    }
 
-      if (forbiddenRequestHeaders.has(lcHeader) || lcHeader.startsWith("sec-") || lcHeader.startsWith("proxy-")) {
+    const keys = Object.keys(flag.requestHeaders);
+    let n = keys.length;
+    while (n--) {
+      const key = keys[n];
+      if (key.toLowerCase() === lcHeader) {
+        flag.requestHeaders[key] += ", " + value;
         return;
       }
-
-      const keys = Object.keys(flag.requestHeaders);
-      let n = keys.length;
-      while (n--) {
-        const key = keys[n];
-        if (key.toLowerCase() === lcHeader) {
-          flag.requestHeaders[key] += ", " + value;
-          return;
-        }
-      }
-      flag.requestHeaders[header] = value;
-    }
-
-    get _ownerDocument() {
-      return idlUtils.implForWrapper(window.document);
     }
+    flag.requestHeaders[header] = value;
   }
+}
 
-  Object.defineProperty(XMLHttpRequest.prototype, Symbol.toStringTag, {
-    value: "XMLHttpRequest",
-    writable: false,
-    enumerable: false,
-    configurable: true
-  });
+setupForSimpleEventAccessors(XMLHttpRequestImpl.prototype, ["readystatechange"]);
 
-  setupForSimpleEventAccessors(XMLHttpRequest.prototype, ["readystatechange"]);
+Object.defineProperty(XMLHttpRequestImpl, "UNSENT", { value: 0, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl, "OPENED", { value: 1, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl, "HEADERS_RECEIVED", { value: 2, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl, "LOADING", { value: 3, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl, "DONE", { value: 4, enumerable: true });
 
-  addConstants(XMLHttpRequest, {
-    UNSENT: 0,
-    OPENED: 1,
-    HEADERS_RECEIVED: 2,
-    LOADING: 3,
-    DONE: 4
-  });
+Object.defineProperty(XMLHttpRequestImpl.prototype, "UNSENT", { value: 0, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl.prototype, "OPENED", { value: 1, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl.prototype, "HEADERS_RECEIVED", { value: 2, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl.prototype, "LOADING", { value: 3, enumerable: true });
+Object.defineProperty(XMLHttpRequestImpl.prototype, "DONE", { value: 4, enumerable: true });
```

Comment:
Looks like the only place in jsdom that gets the constant from the instance is `lib/jsdom/living/xhr/xhr-sync-worker.js` and `lib/jsdom/living/xhr/xhr-utils.js`. Best to fix those for once and for all.